cashctrl_ledger/ledger.py

```diff
@@ -637,97 +557,15 @@ def modify_ledger_entry(self, entry: pd.DataFrame):
         self._client.post("journal/update.json", data=payload)
         self._client.invalidate_journal_cache()
 
-    def delete_ledger_entry(self, ids: Union[str, List[str]]):
+    def delete_ledger_entries(self, ids: List[str] = []):
         """Deletes a ledger entry from the remote CashCtrl instance.
 
         Args:
-            ids (Union[str, List[str]]): The Id(s) of the ledger entry(ies) to be deleted.
+            ids (List[str]): The Ids of the ledger entries to be deleted.
         """
-        if isinstance(ids, list):
-            ids = ",".join(ids)
-        self._client.post("journal/delete.json", {"ids": ids})
+        self._client.post("journal/delete.json", {"ids": ",".join(ids)})
```

Comment:
Better convert ids to str prior to joining. This also works with other types than str:
```
{"ids": ",".join([str(id) for id in ids])}
```
Instead of:
```
{"ids": ",".join(ids)}
```